pkg/kds/cache/snapshot_test.go

```diff
@@ -15,8 +14,8 @@ import (
 )
 
 var _ = Describe("Snapshot", func() {
-	mustMarshalAny := func(pb proto.Message) *any.Any {
-		a, _ := ptypes.MarshalAny(pb)
+	mustMarshalAny := func(pb proto.Message) *anypb.Any {
+		a, _ := anypb.New(pb)
```

Comment:
This PR doesn't change anything, but the convention is that "must" functions will panic on error, not ignore errors.